nbs/02_match_class.ipynb

```diff
@@ -224,7 +224,9 @@
     "        self.graph: DiGraph = graph\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.                    #TODO: add a check of whether it's an edge or an edge collection # Collections Feature
</pre><p>Looks good</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>